src/app/pages/offline-map/offline-map.page.scss

```diff
@@ -27,4 +27,28 @@ ion-backdrop {
   z-index: 21;
   width: calc(100% - 48px);
   margin: 0 24px;
+}
+
+.package-count {
```

Comment:
Enkel og grei den css-koden her. Fungerer fint og ser bra ut. Ideelt sett burde vi nok brukt globale variabler til farge, kanskje også font-weight og font-size, men vet ikke om det er noe vi kan gjenbruke. Kan sikkert bare ha det sånn her.
Det bør egentlig være ganske høy terskel for å innføre nye farger i skissene/koden for å unngå at det er definert farger veldig mange steder for å unngå en grusom jobb om vi en eller gang skal innføre nattmodus eller nye farger eller lignende (ny nve-stil er jo på trappene...). Egentlig designer sin jobb å huske på dette.

Reply:
Dette er veldig nyttig innspill som jeg lærer av. Har brukt globale variable nå. Håper jeg har brukt dem sånn noenlunde riktig. Den mørke fargen som jeg tok fra Figma fant jeg ikke, så jeg erstattet den med --ion-text-color-heading, som _jeg_ synes så lik nok ut.